en/docs/quick-starts/saml-front-channel-logout.md

```diff
@@ -0,0 +1,102 @@
+# Configure SAML 2.0 Front Channel Logout 
+
+This page guides you through configuring [SAML 2.0 front channel logout](TODO:insert-link-to-concept). This is demonstrated using two versions of a sample application called Spring SAML application. 
+
+----
+If you have your own application, click the button below.
+
+<a class="samplebtn_a" href="../../../guides/login/saml-front-channel-logout"   rel="nofollow noopener">I have my own application</a>
+
+----
+
+## Scenario
+
+The user is required to log in to two different applications, `application1` and `application2`. For convenience and security, the user should be logged out of both the applications when the user attempts to logout from either one of them. 
+
+----
+
+## Download SAML metadata 
+
+TODO: dev-portal-fragment
+
+----
+
+## Configure the sample
+
+1.  Download the latest version of the Spring SAML application from [here](https://repo.spring.io/list/release/org/springframework/security/extensions/spring-security-saml/) and unzip it. This unzipped directory will be referred to as `<SAMPLE_HOME>`.
+2.  Rename the downloaded metadata file obtained from the above step as `wso2_abc.xml`.
+3.  Copy it to `<SAMPLE_HOME>/sample/src/main/resources/metadata`.
+4.  Open the `<SAMPLE_HOME>/sample/src/main/webapp/WEB-INF/securityContext.xml` file. 
+5.  Search for **bean id** with the tag as metadata (`bean id=”metadata”`). 
+6.  Under the `list` tag of the metadata, bean id add the following xml configuration and save the file. Note that this has the link to the metadata file,`wso2_abc.xml`, that we generated and renamed. 
+
+    ```xml 
+    <bean class="org.springframework.security.saml.metadata.ExtendedMetadataDelegate">
+    <constructor-arg>
+    <bean class="org.opensaml.saml2.metadata.provider.ResourceBackedMetadataProvider">          
+    <constructor-arg>             
+    <bean class="java.util.Timer" />          
+    </constructor-arg>          
+    <constructor-arg>             
+    <bean class="org.opensaml.util.resource.ClasspathResource">                
+    <constructor-arg value="/metadata/wso2_abc.xml" />             
+    </bean>          
+    </constructor-arg>          
+    <property name="parserPool" ref="parserPool" />       
+    </bean>    
+    </constructor-arg>    
+    <constructor-arg>       
+    <bean class="org.springframework.security.saml.metadata.ExtendedMetadata" />    
+    </constructor-arg> 
+    </bean>
+    ```
+
+## Deploy the sample application
+
+1.  Open a terminal from the `<SAMPLE_HOME>/sample` location and run the following command.
+
+    ```xml
+    mvn clean install
+    ```
+2.  Upon a successful project build, the `spring-security-saml2-sample.war` file can be found in the `<SAMPLE_HOME>/sample/target` directory. 
```

Comment:
Upon a successful project build --> might be clearer to say "Once the sample has built successfully," 

WDYT?